docs/context/ref/Instrument.md

```diff
@@ -15,13 +15,16 @@ Notes:
 - Not all applications will use the same instrument identifiers, which is why FDC3 allows for multiple to be specified.
 In general, the more identifiers an application can provide, the easier it will be to achieve interoperability.
 
+- The `market` map can be used to further specify the instrument and help achieve interoperability disparate data sources. This is especially useful when using an id that is not globally unique.
```

Comment:
```suggestion
- The `market` map can be used to further specify the instrument and help achieve interoperability between disparate data sources. This is especially useful when using an `id` field that is not globally unique.
```

Could you add an example (e.g. a `id.ticker` field such as...) or a note on why it might not be globally unique (listed on multiple exchanges)